lib/active_record/virtual_attributes/virtual_delegates.rb

```diff
@@ -243,25 +247,30 @@ def self.select_from_alias(to_ref, col, to_model_col_name, src_model_id)
                   to_ref.klass.all
                 end
 
-        src_model   = to_ref.active_record
-        to_table    = select_from_alias_table(to_ref.klass, src_model_id.relation)
-        to_model_id = to_ref.klass.arel_attribute(to_model_col_name, to_table)
-        to_column   = to_ref.klass.arel_attribute(col, to_table)
-        arel        = query.except(:select).select(to_column).arel
-                           .from(to_table)
-                           .where(to_model_id.eq(src_model_id))
+        src_model = to_ref.active_record
+        to_table  = select_from_alias_table(to_ref.klass, src_model_id.relation)
+        if ActiveRecord.version.to_s < "6.1"
+          to_model_id = to_ref.klass.arel_attribute(to_model_col_name, to_table)
+          to_column   = to_ref.klass.arel_attribute(col, to_table)
+        else
+          to_model_id = to_ref.klass.arel_table[to_model_col_name, to_table]
```

Comment:
Is arel_table available on < 6.1?  If so, can we just switch to arel_table everywhere like how it was done in https://github.com/ManageIQ/manageiq-api/pull/1121 ?

Reply:
in 6.1, we want to use arel_table everywhere. there is a deprecation warning if we do not do this.

In 6.0, for the single attribute version, we want to use arel_table. But in the double parameter version, we still want to use `arel_attribute()`

I'll look into possibly providing `arel_attribute` type functionality to `arel_table`.

Since we are kinda dropping 6.0 support, didn't put that much effort into it